docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -11,7 +11,7 @@ include::./attributes.adoc[]
 :sectnums:
 :sectnumlevels: 4
 :toc:
-:extension-status: experimental
+:extension-status: stable
 
 include::./status-include.adoc[]
```

Comment:
I think we just dropped this include for most of the stable extensions. We added it only to non-stable extensions so I would rather just drop it.

Reply:
You need to remove the two `include::./status-include.adoc[]` too :)

Reply:
Thanks. Done